examples/react-fcl-demo/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "fcl-demo",
+  "version": "0.1.0",
+  "dependencies": {
+    "@onflow/fcl": "^0.0.56",
+    "@types/jest": "^26.0.14",
+    "@types/node": "^14.11.3",
+    "@types/react": "^16.9.51",
+    "@types/react-dom": "^16.9.8",
+    "@types/styled-components": "^5.1.3",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-scripts": "3.4.1",
+    "styled-components": "^5.1.0"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "eject": "react-scripts eject",
+    "dev-wallet": "PK=6933b0c557a30513adb402be741464601667f898e61017e58a0be94f593e0f6f fcl-wallet",
```

Comment:
Is there a way we can do this without adding the PK variable. The dev wallet should be able to ask the emulator for this. Just a comment, nothing to be done.